CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h

```diff
@@ -120,8 +124,8 @@ std::unique_ptr<TData> L1ConfigOnlineProdBase<TRcd, TData>::produce(const TRcd&
     if (m_copyFromCondDB) {
       // Get L1TriggerKeyList from EventSetup
       const L1TriggerKeyListRcd& keyListRcd = iRecord.template getRecord<L1TriggerKeyListRcd>();
```

Comment:
there is still a `template` here

Reply:
having a second look, I dont even find where `keyListRcd` is used in the rest of the code. Please remove if not needed